accord-core/src/main/java/accord/messages/ReadData.java

```diff
@@ -61,6 +61,7 @@ public static ReadData create(TxnId txnId, Seekables<?, ?> scope, long executeAt
     private transient boolean isObsolete; // TODO (low priority, semantics): respond with the Executed result we have stored?
     private transient BitSet waitingOn;
     private transient int waitingOnCount;
+    private transient BitSet submitted;
```

Comment:
It's used int `ReadData#read`  to prevent submitting a read to the same command store multiple times. I don't remember the details, but there was definitely a burn test I fixed a while ago where that was happening.

Reply:
Hmm. That shouldn’t be happening, so we should maybe rollback and figure out what’s happening? I can take a look if you have a seed.

Reply:
It was before this and async-chain. I've removed it and put a check on removing command store ids from the waitingOn set